server/v2/cometbft/utils.go

```diff
@@ -206,15 +207,15 @@ func intoABCISimulationResponse(txRes appmanager.TxResult, indexSet map[string]s
 }
 
 // ToSDKEvidence takes comet evidence and returns sdk evidence
```

Comment:
use gogo instead of pulsar here.

Reply:
should we pass the abci type and then do this transition in the consensus module, helps avoid the consensus import in comet. Its not a bad thing to have the import, just a cleanup